lib/transport.ex

```diff
@@ -2,9 +2,11 @@ defmodule Janus.Transport do
   @type reason :: any
   @type state :: any
   @type payload :: map
+  @type keepalive_timeout :: number
 
   @callback connect(any) :: {:ok, state} | {:error, reason}
   @callback send(payload, timeout, state) :: {:ok, state} | {:error, reason, state}
   @callback handle_info(any, state) ::
               {:ok, state} | {:ok, payload, state} | {:error, reason, state}
+  @callback needs_keep_alive?() :: {true, keepalive_timeout} | false
```

Comment:
I think that this could simplifiet. Instead of {:true, value} | false, it could be value | nil. @bblaszkow06 What is your opinion on this?

Reply:
Sounds good. The name requires refactor as well.
1. It's not a good idea to return some value from callback suggesting only boolean
1. `value | nil` would nicely fit callback name `keepalive_timout` or similar

Reply:
BTW, since I'm looking at it: @Qizot DOOOOOOOCS! (for every callback, please)

Reply:
@bblaszkow06 can you check them out now?